README.md

```diff
@@ -27,40 +27,48 @@ I am going to try to populate this README and github issues with modular pieces
 All of the setup is required to run.
 
 ### setup software
+
 - node lts v20 (i use [nvm](https://github.com/nvm-sh/nvm) / [nvm-windows](https://github.com/coreybutler/nvm-windows) to manage my node installs)
 - [pnpm](https://pnpm.io/installation) (performant npm)
 - [docker](https://www.docker.com/) (manages local db + s3 bucket from docker-compose)
 
 ### setup project
+
 - clone project
 - open in vscode or [cursor](https://cursor.sh/)
 - `pnpm install`
 - `docker compose up -d` for mysql and s3
 - copy .env.example to .env `cp .env.example .env`
 
 ### setup mysql db
-- run `pnpm seed`
+
+- run `pnpm db:push` to push the Database schema.
+- run `pnpm db:seed`
 - this might take a few mins so move on
 
 note: this will be missing voicelines, but contain npcs
 
 ### setup clerk
+
 - setup a [clerk](https://clerk.com) account for auth (should take less than 2 mins)
 - update .env with NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY and CLERK_SECRET_KEY
 
 ### setup elevenlabs
+
 - setup an [elevenlabs](https://elevenlabs.io) account for voice generation (need at least the $1 paid tier)
 - update .env with ELEVENLABS_API_KEY
 
 ### setup s3 bucket
+
 - ensure the containers are up from docker compose
-- navigate to http://localhost:9001 and make a bucket and access key
+- run `./bucket-config/apply-bucket-configs.sh`. (if the script is not executable, use `chmod +x ./bucket-config/apply-bucket-configs.sh`)
+- navigate to <http://localhost:9001> and make a bucket and access key
```

Comment:
based on your apply bucket configs, i think the bucket would be required to be named allvoice, we should point that out here.